setup.sh

```diff
@@ -8,20 +8,34 @@ if [ $? -eq 0 ]; then echo "All done loading hadoop"; else echo "Error in loadin
 module load python/3.6.0    # Load python 3.6.0
 if [ $? -eq 0 ]; then echo "All done loading python3.6"; else echo "Error in loading Python" && exit 1; fi
 
-if [ ! -d "/home/wdps1936/elasticsearch-2.4.1" ]; then
+SCRATCH_PATH='/var/scratch2/wdps1936/'
```

Comment:
sure this will work all the time?

Reply:
Yeah! I am creating a directory in the next line (mkdir -p $SCRATCH_PATH) in case the wdps1936 folder is absent.

Reply:
oh 😅 🤦‍♂